### ISSUE TITLE
 Login node fingerprints need to be moved to main SSH instructions

### ISSUE THREAD

**Issue by jkwmoore** [rcgsheffield/sheffield_hpc#1160]
https://docs.hpc.shef.ac.uk/en/latest/troubleshooting.html#login-node-rsa-ecdsa-ed25519-fingerprints

This is a bit hidden out of the way, especially given the importance of it.

I will shift this likely by replication shortly.

**Comment by willfurnass** [comment]
Good idea.  Perhaps some Sphinx macro functionality would allow fingerprints to appear in two places but be defined only once.

**Comment by jkwmoore** [comment]
Yeah we can handle this in the same way I approached https://github.com/rcgsheffield/sheffield_hpc/pull/1151

tl;dr using includes.
